### ISSUE TITLE
[windows] Unable to find event 4 in pending events queue when i m using  Barcode Scanner (FocusNode)

### ISSUE THREAD

**Issue by OusamaDridi** [flutter/flutter#89787]
## Steps to Reproduce

- I m using onfocus with RawKeyboardListener (flutter windows app) when I m using the barcode scanner some event (character) is not captured in my handlekey method ( exp: code.lengtht == 10 but i m getting just 8) , in the console it displays "Unable to find event 4 in pending events queue. Unable to find event 13106 in pending events queue.flutter: RawKeyEvent:" 
- Does anyone have any idea regarding this bug??

**Expected results:** 

**Actual results:** 

<details>
  <summary>Logs</summary>

```
```

```
```

```
```
![image](https://user-images.githubusercontent.com/28295113/132758203-b1530a7d-fbfa-454f-a481-bd8f4ee38cd8.png)

</details>

**Comment by ReniDelonzek** [comment]
Related to https://github.com/flutter/flutter/issues/84210

**Comment by OusamaDridi** [comment]
@ReniDelonzek  no solution found regarding this bug?

**Comment by ReniDelonzek** [comment]
@OusamaDridi not until now

**Comment by danagbemava-nc** [comment]
Since this issue shares a root cause as #84210. 

Closing as Duplicate of #84210 so that it can be tracked in one place.

Thank you
